src/main/webapp/meal-page-script.js

```diff
@@ -55,7 +55,14 @@ function createElementByTag(text, tag) {
   return element;
 }
 
+function redirectToSimilar() {
+    fetch('/meal/similar').then(response => response.json()).then((id) => {
+        const url = '/meal.html?id=' + id.toString();
```

Comment:
You can use a [template literal](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) here.